tests/utils.ts

```diff
@@ -90,6 +90,25 @@ export async function callContract(
   await sendAndReturnFinalized(signer, tx);
 }
 
+export async function rpcContract(
+  api: ApiPromise,
+  dest: Address,
```

Comment:
It's called `contractAddress` in the other util functions, I'd prefer to keep the same name in all of them. 
I named it `contractAddress` to make it more obvious for external people looking at the code what we're referring to, but I'm also fine with using the substrate prop name `dest`. Could you change it in the other functions, too, then?

Reply:
Renamed to contractAddress